pkg/webhooks/workload_webhook_test.go

```diff
@@ -466,6 +510,25 @@ func TestValidateWorkloadUpdate(t *testing.T) {
 				field.Invalid(field.NewPath("spec").Child("priorityClassName"), nil, ""),
 			},
 		},
+		"podSetUpdates should be immutable when state is ready": {
```

Comment:
Add a case for updates in other fields where the podSetUpdates don't change

Reply:
IIUC this test specifically checks for changes in the label field when `podsetupdates` must be immutable. You'd like to see a test that allows us to change fields?

Reply:
I'd like to see a test where `podSetUpdates` don't change, but other fields do. In other words: a test for the case where the validation passes.